returnn/datasets/basic.py

```diff
@@ -414,9 +414,9 @@ def get_seq_order_for_epoch(self, epoch, num_seqs, get_seq_len=None):
         nth = 1
       else:
         nth = int(tmp[1])
-      rnd_seed = ((full_epoch - 1) // nth + 1) if full_epoch else 1
-      rnd = Random(rnd_seed + self.random_seed_offset)
-      rnd.shuffle(seq_index)  # Shuffle sequences.
+      rnd_seed = self._get_random_seed_for_epoch(epoch=epoch, num_epochs_fixed=nth)
+      numpy.random.seed(rnd_seed)
+      seq_index = numpy.random.permutation(num_seqs).tolist()  # type: typing.List[int]
```

Comment:
Using `rnd.shuffle` is more efficient.

Reply:
Ah sorry, this is using `num_seqs` as argument. No, I think then `rnd.permutation` is ok. But not the global Numpy rnd.